cypress/support/commands.ts

```diff
@@ -222,34 +187,45 @@ Cypress.Commands.add('clickFirstFileInFileTree' as keyof Chainable, () => {
   cy.get('[data-cy-debug-tree="root"] > app-tree-item').eq(0).find('app-tree-item').eq(0).click();
 });
 
-Cypress.Commands.add('clickFirstChildInFileTree' as keyof Chainable, () => {
-  cy.get('[data-cy-debug-tree="root"] > app-tree-item app-tree-item').find('div > div').eq(0).click();
-})
-
-Cypress.Commands.add('clickRowInTable', (index: number) => {
+Cypress.Commands.add('clickRowInTable' as keyof Chainable, (index: number) => {
   cy.get('[data-cy-debug="tableBody"]').get('tbody').find('tr').eq(index).click();
 });
 
-Cypress.Commands.add('checkFileTreeLength', (n: number) => {
-  cy.get('[data-cy-debug-tree="root"] > app-tree-item').should('have.length', n);
+Cypress.Commands.add('checkFileTreeLength' as keyof Chainable, (length: number) => {
+  cy.get('[data-cy-debug-tree="root"] > app-tree-item').should('have.length', length);
 });
 
-Cypress.Commands.add('refreshApp', () => {
-  cy.intercept({
-    method: 'GET',
-    hostname: 'localhost',
-    url: /\/api\/*?/g,
-  }).as('apiCall');
+Cypress.Commands.add('refreshApp' as keyof Chainable, () => {
+  interceptGetApiCall(/\/api\/*?/g, 'apiCall');
   cy.get('[data-cy-debug="refresh"]').click();
   cy.wait('@apiCall').then(() => cy.log('All api requests have completed'));
 });
 
-Cypress.Commands.add('getTableBody', () => {
+Cypress.Commands.add('getTableBody' as keyof Chainable, () => {
   return cy.get('[data-cy-debug="tableBody"]').get('tbody');
 });
 
-Cypress.Commands.add('assertDebugTableLength', (length: number) => {
+Cypress.Commands.add('assertDebugTableLength' as keyof Chainable, (length: number) => {
   length === 0
     ? cy.getTableBody().find('tr').should('not.exist')
     : cy.getTableBody().find('tr').should('have.length', length);
 });
+
+function interceptGetApiCall(url: StringMatcher, alias: string): void {
```

Comment:
I see this method is called a lot of times with: `/\/api\/*?/g` as url, maybe it's better to have that as default value for url?